python/podio_gen/generator_utils.py

```diff
@@ -29,6 +29,37 @@ def _prefix_name(name, prefix):
   return name
 
 
+def get_julia_type(cpp_type=None, is_array=False, array_type=None, array_size=None):
+  """Parse the given c++ type to a Julia type"""
+  builtin_types_map = {"int": "Int32", "float": "Float32", "double": "Float64",
+                       "bool": "Bool", "long": "Int64", "unsigned int": "UInt32",
+                       "unsigned long": "UInt64", "char": "Char", "short": "Int16",
+                       "long long": "Int64", "unsigned long long": "UInt64"}
+  # is a global type as described in test_MemberParser.py #L121
+  if cpp_type and cpp_type.startswith("::"):
+    cpp_type = cpp_type[2:]
+  if cpp_type in builtin_types_map:
+    return builtin_types_map[cpp_type]
+
+  if not is_array:
+    if cpp_type.startswith('std::'):
+      cpp_type = cpp_type[5:]
+    if cpp_type in ALLOWED_FIXED_WIDTH_TYPES:
+      regex_string = re.split("(u|)int(8|16|32|64)_t", cpp_type)
+      cpp_type = regex_string[1].upper() + "Int" + regex_string[2]
+      return cpp_type
+
+  else:
+    array_type = get_julia_type(cpp_type=array_type)
```

Comment:
```suggestion
    array_type = get_julia_type(array_type)
```
We can also skip the keyword here, since now it is clear which is the only required argument.

Reply:
I feel we should keep these for the clarity of contributors and maintainers as this will help in easy debugging and understanding of code.